label/models/label_print.py

```diff
@@ -146,6 +127,6 @@ def name_search(self, name='', args=None, operator='ilike', limit=None):
         data = self.env.context.get('model_list')
         if data:
             args.append(('model', 'in', safe_eval(data)))
-        ret_vat = super(IrModelFields, self).name_search(
+        ret_vat = super().name_search(
```

Comment:
Remove the whole class inheritance, I think it's useless. See if it still works.